src/btf.cpp

```diff
@@ -941,28 +915,30 @@ std::set<std::string> BTF::get_all_structs() const
 std::unordered_set<std::string> BTF::get_all_iters_from_btf(
     const struct btf *btf) const
 {
-  std::unordered_set<std::string> iter_set;
-  const std::string prefix = "bpf_iter__";
+  constexpr std::string_view prefix = "bpf_iter__";
```

Comment:
This apparently causes an error on Meta builds (which I think are using clang)
https://github.com/bpftrace/bpftrace/pull/4009